api_tests/nodes/filters/test_filters.py

```diff
@@ -232,6 +236,18 @@ def test_non_mutating_list_filtering_tests(
         assert child_node_one._id not in ids
         assert parent_project._id not in ids
 
+    #   test_child_of
+        url = '{}{}'.format(
+            child_of_url,
+            child_node_two._id
+        )
+        res = app.get(url, auth=user.auth)
+        assert res.status_code == 200
+        assert len(res.json['data']) == 2
+        ids = [node_['id'] for node_ in res.json['data']]
+        assert grandchild_node_two._id in ids
+        assert great_grandchild_node_two._id in ids
+
```

Comment:
The ticket requirements mention that this is needed to "support deleting a node with many descendants in a sane way." Can you add a test for this use case?

Reply:
My understanding is Embosf just needs the data so the user can make a decision about what should be deleted, not actually using this functionality to delete anything. So you could use this filter for anything that needs the guids, etc for the whole node tree. So I don't think there really needs to be an extra test for this if all I'm doing is taking the descendants and bulk deleting them.